src/app/mail/mail-trash-list/mail-trash-list.component.html

```diff
@@ -0,0 +1,234 @@
+                      <div ngbDropdownMenu aria-labelledby="folderDropdownMenuButton">
+                          <a class="dropdown-item" href=""><span class="folder-color-box is-success"></span>Folder
+                              1</a>
+                          <a class="dropdown-item" href=""><span class="folder-color-box is-info"></span>Another
+                              Folder 21</a>
+                          <a class="dropdown-item" href=""><span class="folder-color-box is-warning"></span>Folder 33</a>
+                          <a class="dropdown-item" href="" translate="mail_sidebar.inbox"><i class="icon icon-inbox"></i>Inbox</a>
+                          <a class="dropdown-item" href="" translate="mail_sidebar.draft"><i class="icon icon-inbox"></i>Draft</a>
+                          <a class="dropdown-item" href="" translate="mail_sidebar.archive"><i class="icon icon-archive"></i>Archive</a>
+                          <span class="divider-item d-block"></span>
+                          <a class="dropdown-item" href="" data-toggle="modal" data-target="#createFoldersModal" translate="mail_list.create_folder"><i
+                                  class="icon icon-folder"></i>Create folder</a>
+                      </div>
+                  </div>
+              </li>
+              <li class="btn-dropdown mailbox-dropdown mailbox-dropdown-lg more-options-dropdown pl-0">
+                  <div ngbDropdown class="dropdown">
+                      <button ngbDropdownToggle class="disabledx" role="button" id="moreOptionsDropdownMenuButton" translate="mail_list.more">
+                          More
+                      </button>
+                      <div id="moreOptionsDropdownMenu" ngbDropdownMenu aria-labelledby="moreOptionsDropdownMenuButton">
+                          <a class="dropdown-item" href="" translate="mail_list.add_star">Add star</a>
+                          <a class="dropdown-item" href="" translate="mail_list.mark_as_read">Mark as read</a>
+                          <a class="dropdown-item" href="" translate="mail_list.mark_as_unread">Mark as unread</a>
+                      </div>
+                  </div>
+              </li>
+          </ul>
+      </div>
+
+      <div class="col-sm-4 mailbox-actionbar-col hidden-xs-down">
+          <div class="mail-controls d-flex align-items-center justify-content-end">
+              <div class="mail-result-text text-muted">
+                  <small>1-50 of 2346</small>
+              </div>
+              <div class="mail-nav-control">
+                  <button role="button" class="mail-nav-control-prev"><i class="icon icon-chevron-left"></i></button>
+                  <button role="button" class="mail-nav-control-next"><i class="icon icon-chevron-right"></i></button>
+              </div>
+          </div>
+      </div>
+  </div>
+</div>
+
+<!--Will be used later-->
```

Comment:
@Sage-Brezinsky I asked you to only include the code which was needed, and you still copy pasted blindly, if you seee the below code, it is commented that it will be used later, so there is no use of this code here.
This kind of code might be on other places as well, so please look carefully